servconf.c

```diff
@@ -1048,7 +1053,7 @@ match_cfg_line(char **condition, int line, struct connection_info *ci)
 		    ci->address ? ci->address : "(null)",
 		    ci->laddress ? ci->laddress : "(null)", ci->lport);
 
-	while ((attrib = strdelim(&cp)) && *attrib != '\0') {
+	while ((attrib = strdelim(&cp)) && (attrib != NULL) && *attrib != '\0') { // fix CodeQL SM02311
```

Comment:
I don't think this change is needed because `attrib = NULL` will terminate the while loop and `attrib` pointer will not be dereferenced.